### PR TITLE
improvement(version): 666.development is renamed to x.y.dev

### PR DESCRIPTION
**Pull request by bentsi**

### CODE REVIEW

File: SCYLLA-VERSION-GEN [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 PRODUCT=scylla
-VERSION=666.development
+VERSION=x.y.dev
```
Comment (fruch) [inline SCYLLA-VERSION-GEN:4]: this would break the install of the RPM, this one need to match what's on master exactly.

**Review by fruch — CHANGES_REQUESTED** [review]
this change need to happen in tandem with the change in master. otherwise it won't work

File: SCYLLA-VERSION-GEN (hunk 1 above)
Comment (yaronkaikov) [reply SCYLLA-VERSION-GEN:4]: Why are we using x.y.dev instead of 4.4.dev?  the change first need to be merged in Scylla.

Comment (bentsi) [reply SCYLLA-VERSION-GEN]: updated to 4.4.dev

**Comment by bentsi** [comment]
> this change need to happen in tandem with the change in master. otherwise it won't work

I know, this is the "preparation" PR

**Comment by yaronkaikov** [comment]
@bhalevy will send a patch do the same change in Scylla

**Review by yaronkaikov — APPROVED** [review]

**Comment by yaronkaikov** [comment]
https://github.com/scylladb/scylla/pull/7541 Scylla PR was merged

**Comment by yaronkaikov** [comment]
@bentsi Please merge it

**Comment by bentsi** [comment]
merged.

**Pull request merged**